### PR TITLE
Fix when nullResource returned from Transformers

### PR DESCRIPTION
**Pull request by RTLer**
This fix is needed in case of returning NullResource and return null.
In case of meta is not empty we will return meta array

### CODE REVIEW

**Comment by greydnls** [comment]
I need test coverage on this before I can review it.

**Comment by RTLer** [comment]
sorry for the delay @greydnls 
i'm added tests for the nullResource fix

**Comment by amiri27** [comment]
@greydnls any chance to review this pull request?

File: src/Serializer/ArraySerializerWithNull.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+namespace League\Fractal\Serializer;
+
+class ArraySerializerWithNull extends ArraySerializer
```
Comment (greydnls) [inline src/Serializer/ArraySerializerWithNull.php]: I'm not sure we need a separate class for this. Can you walk me through your reasoning for not just adding `null()` to `ArraySerializer`?

Comment (RTLer) [inline src/Serializer/ArraySerializerWithNull.php]: that serializer meant to demonstrate null():null, but i created file in wrong place.
i moved it up to test/stub directory
i don't wanted to make new ArraySerializer

**Comment by amiri27** [comment]
Well done @RTLer ..

**Comment by amiri27** [comment]
@greydnls any chance to merge this?

**Comment by amiri27** [comment]
@greydnls Ping..

**Comment by amiri27** [comment]
@greydnls this behavior is not acceptable for opensource community.. We are waiting more than 25 days for a simple PR..
 These kinds of attention to the community would result more contributions on main code and features..

Hope that my comment didn't bother you..

Regards,

**Comment by amiri27** [comment]
Still after 1 month this PR is open... That's not acceptable behavior... Hope that you revise your attention to the community..

**Comment by Art4** [comment]
Looks like this project has fallen asleep. To sad... 😕

**Comment by martindilling** [comment]
@greydnls Would it be possible to get this merged soon? :)

And @amiri27 I know it's annoying to wait for this kind of thing, but chill, this is an open source project aka. made available for free and made/managed by people in their free time. So it is annoying yes, but I would not call it unacceptable behaviour, life can get in the way for everyone from time to time ;)

**Comment by amiri27** [comment]
Thanks a lot greydnls and @martindilling . I just sent that comment regarding the fix was very small and was covered by good testcase also. Encouraging the community by accepting pull requests faster would cause to put more energy to send better and better features and enhancements.

I'm so sorry again about my last comment and I hope your life goes well.

Regards,

**Pull request merged**